package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@careevolution/mydatahelps-ui",
-  "version": "1.7.1",
+  "version": "1.8.0",
```

Comment:
I think typically would merge and then do npm version / push on main to get a new release, not do it as part of the PR, but not sure it matters